.github/workflows/node.js.yml

```diff
@@ -15,16 +15,15 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [18.x, 20.x]
+        node-version: [22.x]
```

Comment:
hmm i see the small benefit of no longer using the --experimental-modules flag, but I would argue to keep this flexible and allow all node versions 18+ since this is consumer facing and installing a different version of node might be beyond some ppls technical prowess.